csharp/src/Ice/LocalException.cs

```diff
@@ -263,17 +341,17 @@ public InvocationCanceledException(System.Exception ex) : base("", ex)
     /// A generic exception base for all kinds of protocol error
     /// conditions.
     /// </summary>
-    public class ProtocolException : System.Exception
+    public class ProtocolException : Exception
     {
         public string Reason;
```

Comment:
Do we still need this field, can we just use Message?

Reply:
We don't need it, I just did not refactor these exceptions yet. The refactored exceptions are those makes Serializable.